app/assets/javascripts/modules/repo-description.js

```diff
@@ -1,62 +1,75 @@
 moj.Modules.repoDescription = {
```

Comment:
This module name doesn't really reflect what it does any more

Reply:
Very good point.